src/Entities/EnderCrystal.cpp

```diff
@@ -21,9 +23,49 @@ cEnderCrystal::cEnderCrystal(Vector3d a_Pos, bool a_ShowBottom):
 
 
 
+void cEnderCrystal::SetShowBottom(bool a_ShowBottom)
+{
+	m_ShowBottom = a_ShowBottom;
+	if (m_World != nullptr)
```

Comment:
I don't think this can ever happen, entities are always spawned in worlds

Reply:
On  loading the entity this is nullptr

Reply:
Good point, then this probably means most of the other entity property setters are bugged because they don't broadcast anything >.>

Loading is easy though, could you move these properties to be settable via the constructor?

Reply:
I think so.

Yes this is doable.